certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -333,11 +337,23 @@ DEFINE
  TARGETSHARETYPE           = NUMBER
  TARGETIRSCODE             = NUMBER
  TMPVAL                    = NUMBER
- END [DEFINE]
+
+[* TKainz - added 03/14/24
+*]
+ PARAMXFER3HANDLING        = NUMBER
+ STRANSFER3COUNT           = NUMBER
+ STRANSFER3COUNTMAX        = 99
```

Comment:
Do we need this separate `STRANSFER3COUNTMAX` vs. just having the one `STRANSFERCOUNTMAX`?

Reply:
"Need it"? nope.  Just thought it keep the code a bit 'cleaner.  Removed.